### DISCUSSION TITLE
docs: add react-router on routing.mdx

### DISCUSSION THREAD

**Discussion by Maicarons2022** [wailsapp/wails#1754] [Ideas]
File: [`docs/guides/routing.mdx`](https://github.com/wailsapp/wails/blob/master/website/docs/guides/routing.mdx)

Hello developers,  
Seeing that the `routing.mdx` has no `React` item, I wish to add the `react-router`.

__Example__:
```jsx
import { BrowserRouter as Router, Routes, Route } from "react-router-dom";

...

 <Router basename={'/'}>

...

      <Routes>
        <Route path="/" element={<Page0/>}  exact />
        <Route path="/page1" element={<Page1/>} />
        <Route path="/page2" element={<Page2/>} />
...
      </Routes>
</Router>
```

Regrad.

**Comment by leaanthony** [comment]
Thanks! Feel free to raise a PR
